client/js/views/MyProfile.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-container class="profile">
+    <h1>{{ user }} Profile Settings</h1>
+    <br />
+    <v-text-field
+      v-model="email"
+      outlined
+      :dense="$vuetify.breakpoint.mdAndDown ? true : false"
+      hint="email"
+      data-cy="email"
+      autocomplete="email"
+      placeholder="email"
+      @keydown.enter.prevent="submitEmail"
+    />
+    <v-snackbar
+      v-model="showSnackBar"
+      :color="colorValue"
+      content-class="d-flex justify-space-between align-center"
+      data-cy="auth-snackbar"
+    >
+      {{ snackBarMessage }}
+      <v-icon data-cy="close-snackbar" @click="clearSnackBar"> mdi-close </v-icon>
+    </v-snackbar>
+    <!-- <h3>Current Email: test</h3> -->
+  </v-container>
+</template>
+<script>
+export default {
+  name: 'MyProfile',
+  data() {
+    return {
+      user: this.$store.state.auth.username,
+      email: '',
```

Comment:
Once this is hooked up in the full stack we'll want to initialize this value from the user's existing email in case it's already submitted